src/app/logic/ComputerPlayer.js

```diff
@@ -0,0 +1,18 @@
+import getRandomInt from '../utils/getRandomInt';
+
+export const ComputerPlayer = () => {
+  const player = {
+    askQuestion(question, callback) {
+      if (!question) throw new Error('There is no question');
+      callback(question);
```

Comment:
What callback could be used for question? What happens after the question is asked?